server.js

```diff
@@ -136,15 +140,7 @@ function Iterator (down, req, encode) {
   var self = this
 
   this.batch = req.batch || 0
-
-  if (req.options) {
-    if (req.options.gt === null) req.options.gt = undefined
-    if (req.options.gte === null) req.options.gte = undefined
-    if (req.options.lt === null) req.options.lt = undefined
-    if (req.options.lte === null) req.options.lte = undefined
-  }
-
-  this._iterator = down.iterator(req.options)
+  this._iterator = down.iterator(cleanRangeOptions(req.options))
```

Comment:
Because `null` and `undefined` are valid range options in latest `abstract-leveldown`, we must strip them from the options object.